docs/src/docs/next-steps.md

```diff
@@ -16,18 +16,19 @@ If you need a quick way to sign a ModelKit you can follow the same instructions
 
 ## Making your own Kitfile
 
-A Kitfile is the configuration document for your ModelKit. It's written in YAML so it's easy to read. There are four main parts to a Kitfile:
+A Kitfile is the configuration document for your ModelKit. It's written in YAML so it's easy to read. There are five main parts to a Kitfile:
 
 1. The `package` section: Metadata about the ModelKit, including the author, description, and license
-1. The `code` section: Path and information about codebases related to the project, including Jupyter notebook folders
-1. The `datasets` section: Path and information on included datasets
-1. The `model` section: Path and information on the serialized model
+1. The `model` section: Information about the serialized model
+1. The `docs` section: Information about documentation for the modelkit
```

Comment:
ModelKit is with a capital letter rest of the document. Let's continue to follow that.

Reply:
Fixed